.github/workflows/draw-zmk.yml

```diff
@@ -47,53 +52,78 @@ jobs:
       contents: write
 
     steps:
-    - name: Checkout
-      uses: actions/checkout@v3      
-    - name: Install keymap-drawer (pypi)
-      if: inputs.install_branch == ''
-      run: python3 -m pip install keymap-drawer
-    - name: Install keymap-drawer (git)
-      if: inputs.install_branch != ''
-      run: python3 -m pip install "git+https://github.com/caksoylar/keymap-drawer.git@${{ inputs.install_branch }}"
-    - name: Draw keymaps
-      run: |
-        get_args() {
-            local keyboard=$2
-            eval set -- "$1"
-            for arg; do
-                local key=${arg%%:*}
-                local val=${arg#*:}
-                if [ "$key" = "$keyboard" ]; then
-                    echo "$val"
-                    break
-                fi
-            done
-        }
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          # So the reference to the parent commit is available when amending
+          # See:
+          # - https://github.com/stefanzweifel/git-auto-commit-action#using---amend-and---no-edit-as-commit-options
+          # - https://github.com/stefanzweifel/git-auto-commit-action/issues/159#issuecomment-845347950
+          # - https://github.com/actions/checkout
+          fetch-depth: ${{ (inputs.amend_commit == true && 2) || 1 }}
```

Comment:
I know the `== true` and parentheses aren't technically required here, but I prefer to err on the side of caution and readability over terseness. E.g. I want the input to explicitly be the boolean `true`, and ignore any type coercion shenanigans.

Reply:
Sounds good! I am not a fan of the type system of this thing either.